docs/root/start/sandboxes/golang-http.rst

```diff
@@ -97,5 +97,5 @@ The output from the ``curl`` command below should include the body that contains
       Further information about the Envoy Go filter.
    :ref:`Go extension API <envoy_v3_api_file_contrib/envoy/extensions/filters/http/golang/v3alpha/golang.proto>`
       The Go extension filter API.
-   :repo:`Go plugin API <contrib/golang/filters/http/source/go/pkg/api/filter.go>`
+   :repo:`Go plugin API <https://github.com/envoyproxy/envoy/blob/main/contrib/golang/common/go/api/filter.go>`
```

Comment:
use the relative link as above - the `:repo:` role should handle the links (altho annoyingly its currently broke for PRs)

Reply:
also notice this point, all `:repo:` links seem not working in pre-rendered docs

Reply:
yeah, not sure how to resolve

i broke it when optimizing the PR docs build - iirc its not easy/possible to resolve without undoing that optimization